script.sh

```
@@ -0,0 +1,67 @@
+# Build package
+sui client build
+
+# Publish package
+sui client publish --gas-budget 100000000
+
+# Save returned values
+export PACKAGE_ID=<package_id_from_publish>
```

Shellcheck (0.10.0)

[error] Couldn't parse this variable assignment. Fix to allow more checks.

(SC1073)